docs/reference/ocm_sign_hash.md

```diff
@@ -18,19 +18,17 @@ ocm sign hash <private key file> <hash> [<issuer>]
 
 ### Description
 
-
 Print the signature for a dedicated digest value.
 	
-
 ### Examples
 
-```
+```yaml
```

Comment:
```suggestion
```bash
```

why is this yaml syntax? shouldn't this be `bash`?

Reply:
I've added now a new annotation [ExampleCodeStyle](https://github.com/open-component-model/ocm/pull/871/commits/c5817f928554f5a45ae75dbd68f35a3c0bdc479b) to the cobra-commands, which will be used for the code-blocks.